### PR TITLE
Adding basic guides

### PR DESCRIPTION
**Pull request by sharkparty**

### CODE REVIEW

**Comment by hannahmshin** [comment]
only thing i would ask for is maybe a list of things to run before a pr/merge (make sure tests pass, linting, styling, etc). would prob reduce churn on code reviews.

**Comment by sharkparty** [comment]
Good note. 1s

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,5 +1,443 @@
...
+It will fail on any lint error.
+
+### `yarn pretty`
+
+Runs prettier over the entire project to make all the code uniform.
+
+There's also a pre-commit hook if you're committing from the command line.
+
+
+## Testing
```
Comment (sharkparty) [inline CONTRIBUTING.md:69]: @hannahmkim

**Comment by hannahmshin** [comment]
nice looks good to me

**Pull request merged**